Revista.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require_once "Autor.php";
+require_once "Libro.php";
+require_once "IPublicable.php";
+
+class Revista extends Libro implements IPublicable {
```

Comment:
```suggestion
class Revista implements IPublicable {
```
Revista solo debe implementar la interfaz IPublicable